R/edit-spreadsheets.R

```diff
@@ -0,0 +1,173 @@
+#' Create a new spreadsheet
+#' 
+#' Create a new (empty) spreadsheet in your Google Drive. The new spreadsheet
+#' will contain 1 default worksheet titled "Sheet1".
+#' 
+#' @param title the title for the new spreadsheet
+#' 
+#' @export
+new_ss <- function(title)
+{
+  dat <- data.frame("title" = title, 
+                    "mimeType" = "application/vnd.google-apps.spreadsheet")
+  
+  the_body <- jsonlite::toJSON(dat) %>% stringr::str_sub(., 2, -2)
+  
+  the_url <- "https://www.googleapis.com/drive/v2/files"
+  
+  gsheets_POST(the_url, the_body)
+  
+  message(paste0('Spreadsheet "', title, '" created in Google Drive.'))
+}
+
+
+#' Move a spreadsheet to trash{"__src__":"file","ids":"1PmCRtmcX4r7LOS-XVw6u8f-E3sVnFgamJaKCmVF3Sv0","parentId":"0AFlKxBjVWN80Uk9PVA"}
+#' 
+#' Move a spreadsheet to trash in Google Drive.
+#' 
+#' @param x the title or key of a spreadsheet
+#' 
+#' @note Using the key is useful when there are spreadsheets with the same name 
+#' since the default is to send the most recent spreadsheet to the trash.
+#' 
+#' @note Shared spreadsheets can not be removed from your Google Drive with 
```

Comment:
You can delete spreadsheets that are shared with others. As long as you are the owner, you can delete it.